client/source/components/RR/ReviewList/AddReview/AddReview.jsx

```diff
@@ -0,0 +1,184 @@
+    event.preventDefault();
+    requests
+      .post('/reviews', formInput)
+      .then((results) => {
+        console.info(results.status);
+        onClose(event);
+        clearForm();
+      })
+      .catch((err) => console.error(err));
+  };
+
+  return (
+    <Form onSubmit={handleSubmit}>
+      <div className="form-section">
+        <div className="form-section-header">Overall rating *</div>
+        <StarRating rating={formInput.rating} onClick={handleStars} />
+      </div>
+
+      <div className="form-section">
+        <div className="form-section-header">Do you recommend this product?*</div>
+        <div className="radio-options">
+          <label htmlFor="yes">
+            <input
+              name="recommend"
+              type="radio"
+              value={1}
+              id="yes"
+              onChange={handleRecommended}
+              required={true}
+            />
+            Yes
+          </label>
+          <label htmlFor="no">
+            <input
+              name="recommend"
+              type="radio"
+              value={0}
+              id="no"
+              onChange={handleRecommended}
+              required={true}
+            />
+            No
+          </label>
+        </div>
+      </div>
+      <div className="form-section">
+        <div className="form-section-header">Characteristics *</div>
+        <div className="radio-options">
+          {traits.map((trait) => (
+            <Characteristic
+              trait={trait}
+              postID={postID}
+              key={trait[0]}
+              onChange={handleRadioChange}
+            />
+          ))}
+        </div>
+      </div>
+      <FormInput
+        labelText="Review summary"
+        type="text"
+        name="summary"
+        value={formInput.summary}
+        placeholder="Example: Best purchase ever!"
+        onChange={handleInputChange}
+      />
+      <FormTextarea
+        labelText="Review body *"
+        name="body"
+        value={formInput.body}
+        minLength="50"
+        placeholder="Why did you like the product or not?"
+        required={true}
+        disclaimerText={50 - formInput.body.length > 0
+          ? `Minimum characters left: ${50 - formInput.body.length}`
+          : 'Minimum reached'}
+        onChange={handleInputChange}
+      />
+
+      {/* <div className="form-section">
+        <div className="form-section-header">Upload your photos</div>
+      </div> */}
+
+      <FormInput
+        labelText="What is your nickname *"
+        type="text"
+        name="name"
+        value={formInput.name}
+        placeholder="Example: jackson11!"
+        required={true}
+        disclaimerText="For privacy reasons, do not use your full name or email address"
+        onChange={handleInputChange}
+      />
+      <FormInput
+        labelText="Your email *"
+        type="email"
+        name="email"
+        value={formInput.email}
+        placeholder="Example: jackson11@email.com!"
```

Comment:
maybe remove the '!' at the end of the placeholder email address since that's not how email addresses are usually formatted?